tests/samples/0084/code.ara

```diff
@@ -0,0 +1,61 @@
+use Async;
+use Await;
+use Concurrently;
+
+use function async;
+use function await;
+use function concurrently;
+
+class Async {}
+class Await {}
+class Concurrently {}
+
+// async, await, and concurrently operations.
+function async(): void {
+    $a = async bar();
+    $b = await $a;
+    ($c, $d) = concurrently {
+        await baz(),
+        await qux(),
+    };
+}
+
+// async, await, and concurrently function calls.
+function await(): void {
+    $a = async(bar());
+    $b = await($a);
+    ($c, $d) = concurrently(
+        await(baz()),
+        await(qux()),
+    );
```

Comment:
this means using parenthesized expressions with `await` and `async` is now impossible.

since we can't tell if `await ( $foo ) ` is a function call, or `await` operation on `( $foo )`, going with a function call is a safe bet, we can add a rule in the linter to check for call to undefined function named `async` or `await`, which will explain this.